test/visualTest.ts

```diff
@@ -223,168 +314,280 @@ describe("Sunburst", () => {
                 [
                     defaultDataViewBuilder.RegionsDataSet
                 ]);
-
-            if (dataView?.matrix?.rows?.root?.children) {
-            dataView.matrix.rows.root.children = [];
+            dataView.matrix!.rows.root.children = [];
 
             visualBuilder.updateRenderTimeout(
                 dataView,
                 () => {
                     expect(visualBuilder.slices.length).toBe(0);
                     done();
-                });
-            }
+                },
+                2,
+                DefaultWaitForRender);
         });
```

Comment:
drop these 2 arguments in that file

Reply:
Done: [fafcb2a](https://github.com/microsoft/powerbi-visuals-sunburst/pull/95/commits/fafcb2aaf1efb4e1ce60b93d91592fcaa75ec29d)